frontend/src/v5/ui/components/shared/modalsDispatcher/templates/infoModal/infoModal.component.tsx

```diff
@@ -37,10 +37,7 @@ export const InfoModal = ({
 		id: 'infoModal.action.primaryDefault',
 		defaultMessage: 'Ok, close window',
 	}),
-	secondaryButtonLabel = formatMessage({
-		id: 'infoModal.action.secondaryDefault',
-		defaultMessage: 'Go back to Teamspace',
-	}),
+	secondaryButtonLabel,
```

Comment:
Im worried this could cause bugs in other components which uses this modal.

Reply:
I checked and this model is only used by one other component (`checkLatestRevisionReadiness`). "Go back to teamspace" should not be a default message for an info modal, so I have that component passing that message as secondaryLabelButton and tested it still works